### PR TITLE
chore: changes to publish as part of every merge to main @W-10054133@

### PR DESCRIPTION
**Pull request by jag-sfdc**
### What does this PR do?
Publish as part of every merge to main

### What issues does this PR fix or reference?
@W-10054133@

### CODE REVIEW

File: contributing/publishing.md [hunk 1]
```diff
@@ -13,60 +15,15 @@ After feature/bug work has been QA'd and closed, it's time to prepare those chan
 
 The salesforcedx-templates project uses a two-branch strategy. Work that is currently under development is committed to the 'develop' branch. The 'main' branch is what's currently in production or is being staged for production.
 
-## Porting Changes
```
Comment (randi274) [inline contributing/publishing.md:16]: Removing the need for porting 👏👏

File: contributing/developing.md [hunk 2]
```diff
@@ -17,10 +17,9 @@ yarn build
 
 ## Branches
 
-- We work in `develop`.
-- Our released (aka. _production_) branch is `main`.
+- We work off of `main` and we release (aka. _production_) branch is `main`.
```
Comment (randi274) [inline contributing/developing.md:20]: This is a good note - will be be deleting `develop` after this change is complete?

**Review by randi274 — APPROVED** [review]
One small question, but looks good if the answer to that is yes!

File: contributing/developing.md (hunk 2 above)
Comment (jag-sfdc) [reply contributing/developing.md:20]: @randi274 Yes. We already did that for lightning-server to test it works. Will do the same here.

**Pull request merged**